tests/components/image/test_init.py

```diff
@@ -323,15 +331,24 @@ async def _wrap_async_get_still_stream(*args, **kwargs):
             assert not resp.closed
             assert resp.status == HTTPStatus.OK
 
-            mock_image.image_last_updated = datetime.datetime.now()
+            mock_image.image_last_updated = datetime.now()
             mock_image.async_write_ha_state()
             # Two blocks to ensure the frame is written
             await hass.async_block_till_done()
             await hass.async_block_till_done()
 
+        with patch.object(mock_image, "async_image", return_value=b"") as mock:
+            # Simulate a "keep alive" frame
+            freezer.tick(55)
+            async_fire_time_changed(hass)
+            # Two blocks to ensure the frame is written
+            await hass.async_block_till_done()
+            await hass.async_block_till_done()
+            mock.assert_called_once()
+
         with patch.object(mock_image, "async_image", return_value=None):
-            mock_image.image_last_updated = datetime.datetime.now()
-            mock_image.async_write_ha_state()
+            freezer.tick(55)
+            async_fire_time_changed(hass)
```

Comment:
aiohttp doesn't close synchronously so you'll need one more iteration of the event loop of the test to pass

```
    await close_future
+    await asyncio.sleep(0)
    assert resp.closed
```

But awaiting the close future is enough since we know the function finished. I'd remove the `resp.closed` check as we don't need to test the implementation details of aiohttp as they may change in the future